21/day01/src/main.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
```

Comment:
Create a new directory `day01/tests`for your test and write your test in integration_test.rs. Put your input_test.txt in the directory `day01/input`.